stagewhisper/__main__.py

```diff
@@ -39,33 +43,33 @@ def cli():
     better hints to Gooey for how to render widgets
     """
     parser = GooeyParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    basic_group = parser.add_argument_group("Basic")
-    advanced_group = parser.add_argument_group("Advanced Options")
+    basic_group = parser.add_argument_group("Basic", gooey_options=APPEARANCE_DARK if IS_DARK else APPEARANCE_LIGHT)
```

Comment:
An example of what I think @harrislapiroff suggested:
```suggestion
    basic_group = parser.add_argument_group("Basic", gooey_options=FIELD_COLORS_DARK if IS_DARK else {})
```